GenderPayGap.WebUI/Controllers/Admin/AdminDownloadsController.cs

```diff
@@ -0,0 +1,434 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.IO;
+using System.Linq;
+using CsvHelper;
+using GenderPayGap.Core;
+using GenderPayGap.Core.Classes;
+using GenderPayGap.Core.Interfaces;
+using GenderPayGap.Database;
+using GenderPayGap.Database.Models;
+using GenderPayGap.Extensions;
+using GenderPayGap.WebUI.Models.Admin;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace GenderPayGap.WebUI.Controllers
+{
+    [Authorize(Roles = "GPGadmin")]
+    [Route("admin")]
+    public class AdminDownloadsController : Controller
+    {
+
+        private readonly IDataRepository dataRepository;
+
+        public AdminDownloadsController(IDataRepository dataRepository)
+        {
+            this.dataRepository = dataRepository;
+        }
+
+        [HttpGet("downloads")]
+        public IActionResult Downloads()
+        {
+            int firstReportingYear = Global.FirstReportingYear;
+            int currentReportingYear = SectorTypes.Public.GetAccountingStartDate().Year;
+            int numberOfYears = currentReportingYear - firstReportingYear + 1;
+
+            var viewModel = new AdminDownloadsViewModel
+            {
+                ReportingYears = Enumerable.Range(firstReportingYear, numberOfYears).Reverse().ToList()
+            };
+
+            return View("Downloads", viewModel);
+        }
+        
+        [HttpGet("downloads-new/all-organisations")]
```

Comment:
We can change this to `downloads/all-organisations`

Reply:
Oops, missed one. Thanks